CONTRIBUTING.md

```diff
@@ -45,7 +45,8 @@ with these commands:
 
 ```
 brew uninstall --force foo
-HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source foo
+HOMEBREW_NO_INSTALL_FROM_API=1
+brew install --build-from-source foo
```

Comment:
That's not a missing newline, this is correct

Reply:
right, my mistake, I had another typo in the line causing it to fail and I mistakenly assumed it was due to the newline.